app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,72 @@
+<p style="color: green"><%= notice %></p>
+
+<section>
+  <h2 class='recipe-name'><%= @recipe.name %></h2>
+
+  <div class='recipe-main'>
+    
+    <div class='public-btn-div'>
+      <p class='sub-title'>Preparation time : <%= @recipe.preparation_time %></p>
+      <div class='public-btn-flex'>
+        <p class='sub-title'>Public :</p>
+          <% if @recipe.public %>
+          <i class="fa fa-solid fa-toggle-on" style="font-size:28px;color:green"></i>
+          <% end %>
+          <% if !@recipe.public %>
+          <i class="fa fa-solid fa-toggle-off" style="font-size:28px;color:red;border:none"></i>
+          <% end %>
```

Comment:
- [ ] As the previous reviewer mentioned the public recipe toggle doesn't work, kindly make sure the user can toggle between the public/private recipe when clicking this button as for now it doesn't work 

![Screenshot_20230526_182906](https://github.com/gsmalik030/recipe_app/assets/25227022/fdd6cefa-64eb-4597-8ba8-fc7e3971cb2b)

Reply:
@Ismailco thanks for the review